.github/scripts/mobsf_ci.py

```
@@ -0,0 +1,343 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)